### ISSUE TITLE
Unable to get this to work

### ISSUE THREAD

**Issue by the3rdParty** [scr34m/Signature#2]
I have two addresses associated with a single iCloud+ account, set up through a "custom email domain", as detailed [here](https://support.apple.com/en-gb/guide/icloud/mm0e4339d289/1.0/icloud/1.0).

This gives me two sending and receiving addresses **associated with a single iCloud account** (my normal address  abcde@icloud.com and my custom one 1234@vwxyz.net).   Both of these show up as choices for different sending addresses in the mail composer window. 

I was hoping that this plug in would let me associate specific signatures to each of the sending addresses, and automatically select the correct signature depending on the choice of sending address.  At the moment Mail associates signatures with a sending **_account_** (my iCloud account) and not a specific sending **_address_**.  

There is nowhere in the setup/plug-in that allows me to define which signature should be associated with which address, and I am therefore unable to get the plug-in to work as I expected .

**Comment by scr34m** [comment]
I've not tested this on icloud account need to look at on it.
But the plugin must be visible in preferences general tab when you click on the plugins button at the bottom.

**Comment by the3rdParty** [comment]
Yes, the Signature.mailbundle plugin is visible in preferences / general tab / Manage plug-ins

**Comment by scr34m** [comment]
Do you have multiple sending e-mail addresses under perferencses / accounts and on right panel in the adresses drop down? If yes then if you reply to an e-mail the signature will be selected.

On new email currently you have to select because there was an issue and stopped messing with.
